src/components/NotesMindMap/NotesMindMapComponent.js

```diff
@@ -25,19 +25,49 @@ export class NotesMindMapComponent extends Component {
         keyboard: false,
       },
       edges: {
-        arrows: { to: true },
+        // arrows: { to: true },
+        arrows: {
+          to: {
+            enabled: false,
+          },
+        },
+        color: '#F2F2F2',
+        // dashes: true,
         smooth: true,
       },
+      nodes: {
+        borderWidth: 2,
+        shape: 'box',
+        color: '#BB86FC',
+        margin: 10,
+        font: {
+          color: '#F2F2F2',
+          size: 15,
+          face: 'raleway',
+        },
+      },
+
       groups: {
         children: {
-          color: {
-            background: '#eef',
-            borderWidth: 3,
+          borderWidth: 2,
+          shape: 'box',
+          color: '#5dbcaf',
+          margin: 10,
+          font: {
+            color: '#F2F2F2',
+            size: 15,
+            face: 'raleway',
           },
         },
         parent: {
-          color: {
-            background: '#faa',
+          borderWidth: 2,
+          shape: 'box',
+          color: '#BB86FC',
+          margin: 10,
+          font: {
+            color: '#F2F2F2',
+            size: 15,
+            face: 'raleway',
```

Comment:
We have a separate task to change the mind map rendering engine, and it's in progress: https://github.com/DarthVanger/NeuralNotes/issues/29

I like how the mind map looks with the new styles, but it's a waste of time... You're changing code which is going to be deleted anyway.

Reply:
I saw https://github.com/DarthVanger/NeuralNotes/issues/29 

I do not think it's wasted time because it just some styles, which do not take lot time. Also do not see reasons why `vis.js` can't be used